ilcs_parser/__init__.py

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+import pycrfsuite
+import os
+import re
+import warnings
+from collections import OrderedDict
+import string
+
+LABELS = [
+    'Chapter',
+    'ActPrefix',
+    'Section',
+    'SubSection',
+    'Attempted'
+]
+
+PARENT_LABEL  = 'TokenSequence'               # the XML tag for each labeled string
+GROUP_LABEL   = 'Collection'                  # the XML tag for a group of strings
```

Comment:
Citations?

Reply:
I think I'll go with `CitationCollection` to match [probablepeople](https://github.com/datamade/probablepeople/blob/master/probablepeople/__init__.py#L48) and [usaddress](https://github.com/datamade/usaddress/blob/40fff23afaf0144f6fbcf55c112cc033a638b05b/usaddress/__init__.py#L53).